### ISSUE TITLE
quit

### ISSUE THREAD

**Issue by BoXiao126** [PaddlePaddle/PaddleSpatial#312]

**Comment by BoXiao126** [comment]
I run the command:
python main.py --root_path ./data/ --data_path ETTh1.csv --input_dim 7 --percentage 0.05 --diff_steps 1000 --beta_end 0.1    

then the output is like this :

--------------------------------------------------------------------------------------------------------------------------------------------
(paddle) F:\Generative model\PaddleSpatial-main\research\D3VAE>python main.py --root_path ./data/ --data_path ETTh1.csv --input_dim 7 --percentage 0.05 --diff_steps 1000 --beta_end 0.1
E:\Anaconda3\envs\paddle\lib\site-packages\pkg_resources\__init__.py:121: DeprecationWarning: pkg_resources is deprecated as an API
  warnings.warn("pkg_resources is deprecated as an API", DeprecationWarning)
E:\Anaconda3\envs\paddle\lib\site-packages\pkg_resources\__init__.py:2870: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('mpl_toolkits')`.
Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
  declare_namespace(pkg)
E:\Anaconda3\envs\paddle\lib\site-packages\pkg_resources\__init__.py:2870: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('google')`.
Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
  declare_namespace(pkg)
Args in experiment:
Namespace(root_path='./data/', data_path='ETTh1.csv', target='target', features='MS', freq='t', checkpoints='./checkpoints/', arch_instance='res_mbconv', sequence_length=16, prediction_length=16, percentage=0.05, target_dim=1, input_dim=7, hidden_size=128, embedding_dimension=64, diff_steps=1000, dropout_rate=0.1, beta_schedule='linear', beta_start=0.0, beta_end=0.1, scale=0.1, psi=0.5, lambda1=1, gamma=0.01, mult=1, num_layers=2, num_channels_enc=32, channel_mult=2, num_preprocess_blocks=1, num_preprocess_cells=3, groups_per_scale=2, num_postprocess_blocks=1, num_postprocess_cells=2, num_channels_dec=32, num_latent_per_group=8, num_workers=0, patience=5, itr=5, dim=-1, train_epochs=20, batch_size=16, learning_rate=0.005, weight_decay=0.0, loss_type='kl', use_gpu=True, gpu=0, use_multi_gpu=False, devices='0', detail_freq='t')
Use GPU: cuda:0
W0601 18:17:57.627503 24772 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 11.7, Runtime API Version: 11.7
W0601 18:17:57.636489 24772 gpu_resources.cc:91] device: 0, cuDNN Version: 8.4.
>>>>>>>start training : ETTh1.csv_sl_16_pl16_0_dim-1_scale0.1_diffsteps1000_itr0>>>>>>>>>>>>>>>>>>>>>>>>>>
train 578
val 72
test 159

(paddle) F:\Generative model\PaddleSpatial-main\research\D3VAE>

----------------------------------------------------------------------------------------------------------

It outputs nothing. 
It quits at the ‘’denoise_net”, line123 at the exp_model.py

how to solve this issue?

**Comment by ramber1836** [comment]
No clues on what happened. You can inject more stubs to print the detailed debug threads.
